manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
     containerImage: quay.io/openshift/origin-cluster-logging-operator:latest
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
-    olm.skipRange: ">=4.1.0 <4.1.2"
+    olm.skipRange: ">=4.1.0 <4.1.0"
```

Comment:
no version could ever match this range, i know this is just the placeholder, but is it going to cause the semver library that OLM is using to be upset, or is it not going to care. I just don't want to break the logging CI if OLM will wind up choking on this.

Reply:
@ecordell can you weigh in?

Reply:
```go
import (
	"github.com/blang/semver"
	"github.com/stretchr/testify/require"
)

func TestBlang(t *testing.T) {
	r, err := semver.ParseRange(">=4.1.0 <4.1.0")
	require.NoError(t, err)
	v, err := semver.ParseTolerant("4.1.0")
	require.NoError(t, err)
	require.False(t, r(v))
}
```

This test passes, so:

- the range will be parsed with no issues
- 4.1.0 will not match the range

Reply:
so it sounds like this will work as is, however if someone wanted to use it to test upgrading from a true 4.1.0 operator, to a new operator they just built (using this CSV) they could not, they'd have to modify the CSV.

so i guess i'll just change this to "4.1.1" to make everyone's lives easier.

Reply:
well, i take it back...i'd rather not do that unless it becomes necessary.  Since it's not necessary currently, i'm going to suggest we proceed w/ what we've got.  If this skipRange becomes problematic in the future for upgrade testing, we can revisit at that point.